src/marshmallow/fields.py

```diff
@@ -1479,20 +1500,36 @@ def __init__(self, precision: str = SECONDS, **kwargs):
             )
             raise ValueError(msg)
 
+        serde_types = (
+            int,
+            float,
+        )
+
+        if serde_type not in serde_types:
```

Comment:
Nitpick: Either I'd write `if serde_type not in (int, float)` or I'd set `SERDE_TYPES` as class attribute.

I use a class attribute when I expect subclassing. In this case, I don't see much use for subclassing, so both suit me.

Reply:
Cleaned this up to be more inline.